doc/source/index.rst

```diff
@@ -16,7 +16,6 @@ Sections
 
    overview
    api/api
-   api_changes
```

Comment:
Ah, I think if you revert this change and re-add `api_changes` to the docs, then `:orphan:` works (currently the CI is still throwing the same error).
 
> Do you want to keep it for later reference?

We only started to include API changes in the release notes since 0.9. At least I think so. Thing is, I don't want to throw away history like this because of a warning and our CI config.

@moi90 Thanks for keeping up with my requests. :wink: